src/helpers/dateUtils.js

```diff
@@ -2,9 +2,14 @@
 
 export const URGENCY_THRESHOLD = 48;
 
-export const formatDate = (dateString: string) => {
+export const formatDate = (
+  dateString: string,
+  localeUnderscore: string = 'en-EN',
```

Comment:
`en-US`

Reply:
Yes, It's a mistake. I'll change it also. Thanks!